src/test/java/com/google/api/generator/gapic/composer/grpcrest/GrpcRestTestProtoLoader.java

```diff
@@ -82,4 +83,35 @@ public GapicContext parseShowcaseEcho() {
         .setTransport(getTransport())
         .build();
   }
+
+  public GapicContext parseShowcaseWicked() {
```

Comment:
Are we going to need to add an entry in this class for every Showcase client that we generate?

Reply:
We do need an entry in the test loader for every new test proto as the unit tests need to access the protoc generated Java files. 
The name is a little confusing in this test due to historic reasons, but I don't consider this proto a showcase proto that will be used by showcase clients, this proto is used for unit tests only.